src/sdk/translations/ifw_zh_CN.ts

```diff
@@ -878,188 +878,188 @@
-        <translation>无法设置文件“%1”的权限。</translation>
+        <translation>无法设置文件"%1"的权限。</translation>
     </message>
     <message>
         <source>Cannot remove file &quot;%1&quot;: %2</source>
-        <translation>无法移除文件“%1”：%2</translation>
+        <translation>无法移除文件"%1"：%2</translation>
     </message>
     <message>
         <source>Cannot move file &quot;%1&quot; to &quot;%2&quot;: %3</source>
-        <translation>无法将文件“%1”移动到“%2”：%3</translation>
+        <translation>无法将文件"%1"移动到"%2"：%3</translation>
     </message>
     <message>
         <source>Installer at &quot;%1&quot; needs to be an offline one.</source>
-        <translation>“%1”的安装程序需要是脱机安装程序。</translation>
+        <translation>"%1"的安装程序需为脱机安装程序。</translation>
     </message>
     <message>
         <source>Cannot open file &quot;%1&quot; for reading.</source>
-        <translation>无法打开文件“%1”进行读取。</translation>
+        <translation>无法打开文件"%1"进行读取。</translation>
     </message>
     <message>
         <source>Cannot read file &quot;%1&quot;: %2</source>
-        <translation>无法读取文件“%1”：%2</translation>
+        <translation>无法读取文件"%1"：%2</translation>
     </message>
     <message>
         <source>Cannot open file &quot;%1&quot; for reading: %2</source>
-        <translation>无法打开文件“%1”进行读取：%2</translation>
+        <translation>无法打开文件"%1"进行读取：%2</translation>
     </message>
     <message>
         <source>Cannot create target directory: &quot;%1&quot;.</source>
-        <translation>无法创建目标目录：“%1”。</translation>
+        <translation>无法创建目标目录："%1"。</translation>
     </message>
     <message>
         <source>Unknown exception caught: %1.</source>
         <translation>捕获到未知异常：%1。</translation>
     </message>
     <message>
         <source>Removing file &quot;%1&quot;.</source>
-        <translation>正在移除文件“%1”。</translation>
+        <translation>正在移除文件"%1"。</translation>
     </message>
     <message>
         <source>Cannot remove file &quot;%1&quot;.</source>
-        <translation>无法移除文件“%1”。</translation>
+        <translation>无法移除文件"%1"。</translation>
     </message>
     <message>
         <source>Cannot remove directory &quot;%1&quot;: %2</source>
-        <translation>无法移除目录“%1”：%2</translation>
+        <translation>无法移除目录"%1"：%2</translation>
     </message>
 </context>
 <context>
     <name>QInstaller::CreateShortcutOperation</name>
     <message>
         <source>&lt;target&gt; &lt;link location&gt; [target arguments] [&quot;workingDirectory=...&quot;] [&quot;iconPath=...&quot;] [&quot;iconId=...&quot;] [&quot;description=...&quot;]</source>
-        <translation>&lt;目标&gt; &lt;链接位置&gt; [target arguments] [&quot;workingDirectory=...&quot;] [&quot;iconPath=...&quot;] [&quot;iconId=...&quot;] [&quot;description=...&quot;]</translation>
+        <translation>&lt;target&gt; &lt;link location&gt; [target arguments] ["workingDirectory=..."] ["iconPath=..."] ["iconId=..."] ["description=..."]</translation>
     </message>
```

Comment:
Translations got removed?
`target`, `link location`

Reply:
These translations were removed on purpose by the translators